README.md

```diff
@@ -31,7 +31,7 @@ This is a collection of links and cannot guarantee that all information is up to
 | WPI HPRC               | Worcester Polytechnic Institute              | IREC                                                         | [https://impactapi.causeview.com/Thunder/actionpage/c005859a-ffad-4b9d-8d0c-f414096506cd](https://impactapi.causeview.com/Thunder/actionpage/c005859a-ffad-4b9d-8d0c-f414096506cd)<br>To have the donation reach our club please set the "Designation" dropdown to "Other" and enter "WPI AIAA (281-AG) for WPI HPRC" | [https://aiaa.wpi.edu/hprc/](https://aiaa.wpi.edu/hprc/)     | [@wpi_hprc](https://www.instagram.com/wpi_hprc/)             |
 | MSU Rocketry Club      | Morehead State                               | Balloons, solid rockets, liquid engine                       | [https://securelb.imodules.com/s/1869/18/interior.aspx?sid=1869&gid=2&pgid=418&cid=1063&dids=640](https://securelb.imodules.com/s/1869/18/interior.aspx?sid=1869&gid=2&pgid=418&cid=1063&dids=640) |                                                              |                                                              |
 | Olin Rocketry      | Olin College of Engineering                               | Spaceport America Cup, Solid Rocket Engine Development                       | [https://www.olinrocketry.com/donate](https://www.olinrocketry.com/donate) | [https://olinrocketry.com/](https://olinrocketry.com/) | [@olinrocketry](https://www.instagram.com/olinrocketry/) |
-
+| STES Rocketry      | Sinhgad Technical Education Society, Pune-India                               | Spaceport America Cup, EuRoC, L2 Solid Rocket Engine Development, L3 Hybrid Engine Development, Plant-based composites, STEM outreach for geographically challenged schools.                        | [PayPal: ](https://www.paypal.com/paypalme/stesrocketry?country.x=IN&locale.x=en_GB) | [https://stesrocketry.in](https://stesrocketry.in) | [@stesrocketry](https://www.instagram.com/stesrocketry/) |
```

Comment:
I attempted to verify the paypal link by sending money but received an error message. 
"Sorry, this recipient can’t accept payments right now."
Let me know when this is resolved and once the Paypal account is able to accept money I will merge this in.

Reply:
Hi again, 
I'm checking in to see if things have been resolved regarding accepting payments. Should I give it another try?